huaweicloud/services/dew/resource_huaweicloud_csms_secret.go

```diff
@@ -308,7 +308,7 @@ func resourceCsmsSecretImport(ctx context.Context, d *schema.ResourceData, _ int
 	return []*schema.ResourceData{d}, nil
 }
 
-func parseID(id string) (string, string) {
+func parseID(id string) (str1 string, str2 string) {
 	parts := strings.SplitN(id, "/", 2)
```

Comment:
SplitN should not be used
```suggestion
	parts := strings.Split(id, "/")
```

Reply:
fixed